packages/sdk/README.md

```diff
@@ -0,0 +1,5 @@
+# @cosmwasm/sdk
+
+[![npm version](https://img.shields.io/npm/v/@cosmwasm/sdk.svg)](https://www.npmjs.com/package/@cosmwasm/sdk)
+
+An SDK to build CosmWasm clients.
```

Comment:
A good start. We will definitely need better docs, but let's get this working for us first